common/lib/xmodule/xmodule/modulestore/xml.py

```diff
@@ -339,9 +339,15 @@ class to use if none is specified in entry_points
         # that have a course.xml. We sort the dirs in alpha order so we always
         # read things in the same order (OS differences in load order have
         # bitten us in the past.)
+
+        if self.library:
+            self.parent_xml = 'library.xml'
```

Comment:
Also, rather than passing `library` as a boolean argument, it might be better to have two subclasses with the relevant constants defined.